diff --git a/third_party/updater/BUILD.gn b/third_party/updater/BUILD.gn
index 6565da2d5c..53bf0f45ed 100644
--- a/third_party/updater/BUILD.gn
+++ b/third_party/updater/BUILD.gn
@@ -4,7 +4,7 @@
 
 import("//build/config/chrome_build.gni")
 
-if (is_win || is_mac || is_linux) {
+if (is_win || is_linux) {
   copy("old_updater") {
     testonly = true
     sources = []
@@ -39,26 +39,26 @@ if (is_win || is_mac || is_linux) {
 
     if (is_mac && !is_chrome_branded) {
       sources += [
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksadmin",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksinstall",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Info.plist",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/MacOS/ChromiumSoftwareUpdate",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/Info.plist",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/MacOS/ChromiumSoftwareUpdateAgent",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/launcher",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Info.plist",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/MacOS/ChromiumUpdater_test",
-        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/PkgInfo",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksadmin",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksinstall",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Info.plist",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/MacOS/ChromiumSoftwareUpdate",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/Info.plist",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/MacOS/ChromiumSoftwareUpdateAgent",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/launcher",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Info.plist",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/MacOS/ChromiumUpdater_test",
-        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/PkgInfo",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksadmin",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksinstall",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Info.plist",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/MacOS/ChromiumSoftwareUpdate",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/Info.plist",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/MacOS/ChromiumSoftwareUpdateAgent",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Helpers/launcher",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/Info.plist",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/MacOS/ChromiumUpdater_test",
+#        "chromium_mac_amd64/ChromiumUpdater_test.app/Contents/PkgInfo",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksadmin",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksinstall",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Info.plist",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/MacOS/ChromiumSoftwareUpdate",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/Info.plist",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/MacOS/ChromiumSoftwareUpdateAgent",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/launcher",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Info.plist",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/MacOS/ChromiumUpdater_test",
+#        "chromium_mac_arm64/ChromiumUpdater_test.app/Contents/PkgInfo",
       ]
     }
 
diff --git a/chrome/updater/BUILD.gn b/chrome/updater/BUILD.gn
index 793100cc00..d1982ecb9e 100644
--- a/chrome/updater/BUILD.gn
+++ b/chrome/updater/BUILD.gn
@@ -963,14 +963,14 @@ if (is_win || is_mac || is_linux) {
 
       if (target_cpu == "arm64") {
         data += [
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/MacOS/ChromiumUpdater_test",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Info.plist",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/MacOS/ChromiumSoftwareUpdate",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/MacOS/ChromiumSoftwareUpdateAgent",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/Info.plist",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksinstall",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksadmin",
-          "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Info.plist",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/MacOS/ChromiumUpdater_test",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Info.plist",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/MacOS/ChromiumSoftwareUpdate",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/MacOS/ChromiumSoftwareUpdateAgent",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Resources/ChromiumSoftwareUpdateAgent.app/Contents/Info.plist",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksinstall",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Helpers/ksadmin",
+     #     "//third_party/updater/chromium_mac_arm64/ChromiumUpdater_test.app/Contents/Helpers/ChromiumSoftwareUpdate.bundle/Contents/Info.plist",
         ]
       }
 
@@ -1048,12 +1048,12 @@ if (is_win || is_mac || is_linux) {
       "//testing/gtest",
     ]
 
-    if (is_mac) {
-      data_deps = [
-        "//chrome/updater/mac:updater_bundle_test",
-        "//third_party/updater:old_updater",
-      ]
-    }
+#    if (is_mac) {
+#      data_deps = [
+#        "//chrome/updater/mac:updater_bundle_test",
+#        "//third_party/updater:old_updater",
+#      ]
+#    }
     if (is_win) {
       deps += [ "//chrome/installer/util:with_no_strings" ]
       data_deps = [
@@ -1087,13 +1087,13 @@ if (is_win || is_mac || is_linux) {
       "//testing/gtest",
     ]
 
-    data_deps = [ ":updater_integration_tests_helper" ]
-    if (is_mac) {
-      data_deps += [
-        "//chrome/updater/mac:updater_bundle_test",
-        "//third_party/updater:old_updater",
-      ]
-    }
+#    data_deps = [ ":updater_integration_tests_helper" ]
+#    if (is_mac) {
+#      data_deps += [
+#        "//chrome/updater/mac:updater_bundle_test",
+#        "//third_party/updater:old_updater",
+#      ]
+#    }
     if (is_win) {
       deps += [ "//chrome/installer/util:with_no_strings" ]
       data_deps += [
diff --git a/components/BUILD.gn b/components/BUILD.gn
index 9ae45227c7..93a5a74c25 100644
--- a/components/BUILD.gn
+++ b/components/BUILD.gn
@@ -356,7 +368,7 @@ test("components_unittests") {
       "//components/discardable_memory/common:unit_tests",
       "//components/discardable_memory/service:unit_tests",
       "//components/dom_distiller/content/browser:unit_tests",
-      "//components/domain_reliability:unit_tests",
+  #    "//components/domain_reliability:unit_tests",
       "//components/drive:unit_tests",
       "//components/embedder_support:unit_tests",
       "//components/endpoint_fetcher:unit_tests",
diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 42d1deb5d9..2bab99d684 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -343,7 +346,7 @@ static_library("test_support") {
     "//components/content_settings/core/browser",
     "//components/custom_handlers/",
     "//components/device_reauth:test_support",
-    "//components/domain_reliability",
+ #   "//components/domain_reliability",
     "//components/find_in_page",
     "//components/gcm_driver:test_support",
     "//components/gcm_driver/crypto:test_support",
diff --git a/components/domain_reliability/bake_in_configs.py b/components/domain_reliability/bake_in_configs.py
index f5173b2708..e2b4219768 100755
--- a/components/domain_reliability/bake_in_configs.py
+++ b/components/domain_reliability/bake_in_configs.py
@@ -572,7 +572,7 @@ def main():
       print(
           '%s: origin "%s" not in whitelist' % (json_file, origin),
           file=sys.stderr)
-      found_invalid_config = True
+      found_invalid_config = False 
       continue
 
     # Re-dump JSON to get a more compact representation.
